js/client-chat.js

```diff
@@ -1099,6 +1099,16 @@
 					this.$chat.append('<div class="notice">' + Tools.sanitizeHTML(row.slice(1).join('|')) + '</div>');
 					break;
 
+				case 'uhtml':
+					this.$chat.append('<div class="notice" name="' + row[1] + '">' + row.slice(2).join('|') + '</div>');
```

Comment:
Use class instead of name. `div class="notice uhtml-' + toId(row[1]) + `"` etc

Also, `sanitizeHTML`. HTML injection blah blah.